Task/python/Taskfile.yaml

```diff
@@ -0,0 +1,121 @@
+# https://taskfile.dev
```

Comment:
Can we rename to Taskfile.yml?  I will hate my life forever if we don't nip this in the bud